turbo.json

```diff
@@ -15,24 +24,6 @@
       "cache": false,
       "persistent": true
     },
-    "antlr4-c3#build": {
-      "dependsOn": ["antlr4#build"]
-    },
-    "@neo4j-cypher/language-support#build": {
-      "dependsOn": ["antlr4#build", "antlr4-c3#build"]
-    },
```

Comment:
Now when I do `npm run clean --force` and `npm run build` it cannot find antlr4 or antlr4-c3:

```
antlr4:build: cache miss, executing 2607cb7a1baba40d
@neo4j-cypher/language-support:build: cache miss, executing 1d715a04baa6bc36
@neo4j-cypher/language-support:build: 
@neo4j-cypher/language-support:build: > @neo4j-cypher/language-support@2.0.0-next.3 build
@neo4j-cypher/language-support:build: > npm run gen-parser && concurrently 'npm:build-types' 'npm:build-esm' 'npm:build-commonjs'
@neo4j-cypher/language-support:build: 
antlr4:build: 
antlr4:build: > antlr4@4.13.1 build
antlr4:build: > webpack
antlr4:build: 
@neo4j-cypher/language-support:build: 
@neo4j-cypher/language-support:build: > @neo4j-cypher/language-support@2.0.0-next.3 gen-parser
@neo4j-cypher/language-support:build: > antlr4 -Dlanguage=TypeScript -visitor src/antlr-grammar/CypherCmdLexer.g4 src/antlr-grammar/CypherCmdParser.g4 -o src/generated-parser/ -Xexact-output-dir
@neo4j-cypher/language-support:build: 
@neo4j-cypher/language-support:build: warning(109): CypherParser.g4:20:0: options ignored in imported grammar CypherParser
@neo4j-cypher/language-support:build: [build-types] 
@neo4j-cypher/language-support:build: [build-types] > @neo4j-cypher/language-support@2.0.0-next.3 build-types
@neo4j-cypher/language-support:build: [build-types] > tsc --emitDeclarationOnly --outDir dist/types
@neo4j-cypher/language-support:build: [build-types] 
@neo4j-cypher/language-support:build: [build-esm] 
@neo4j-cypher/language-support:build: [build-esm] > @neo4j-cypher/language-support@2.0.0-next.3 build-esm
@neo4j-cypher/language-support:build: [build-esm] > esbuild ./src/index.ts --bundle --format=esm --sourcemap --outfile=dist/esm/index.mjs
@neo4j-cypher/language-support:build: [build-esm] 
@neo4j-cypher/language-support:build: [build-commonjs] 
@neo4j-cypher/language-support:build: [build-commonjs] > @neo4j-cypher/language-support@2.0.0-next.3 build-commonjs
@neo4j-cypher/language-support:build: [build-commonjs] > esbuild ./src/index.ts --bundle --format=cjs --sourcemap --outfile=dist/cjs/index.cjs
@neo4j-cypher/language-support:build: [build-commonjs] 
@neo4j-cypher/language-support:build: [build-esm] ✘ [ERROR] Could not resolve "antlr4"
@neo4j-cypher/language-support:build: [build-esm] 
@neo4j-cypher/language-support:build: [build-esm]     src/helpers.ts:8:7:
@neo4j-cypher/language-support:build: [build-esm]       8 │ } from 'antlr4';
@neo4j-cypher/language-support:build: [build-esm]         ╵        ~~~~~~~~
@neo4j-cypher/language-support:build: [build-esm] 
@neo4j-cypher/language-support:build: [build-esm]   The module "./dist/antlr4.web.mjs" was not found on the file system:
@neo4j-cypher/language-support:build: [build-esm] 
@neo4j-cypher/language-support:build: [build-esm]     ../../node_modules/antlr4/package.json:64:18:
@neo4j-cypher/language-support:build: [build-esm]       64 │         "import": "./dist/antlr4.web.mjs",
@neo4j-cypher/language-support:build: [build-esm]          ╵                   ~~~~~~~~~~~~~~~~~~~~~~~
@neo4j-cypher/language-support:build: [build-esm] 
@neo4j-cypher/language-support:build: [build-esm]   You can mark the path "antlr4" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
@neo4j-cypher/language-support:build: [build-esm] 
@neo4j-cypher/language-support:build: [build-commonjs] ✘ [ERROR] Could not resolve "antlr4"
@neo4j-cypher/language-support:build: [build-commonjs] 
@neo4j-cypher/language-support:build: [build-commonjs]     src/signatureHelp.ts:6:32:
@neo4j-cypher/language-support:build: [build-commonjs]       6 │ import { ParseTreeWalker } from 'antlr4';
@neo4j-cypher/language-support:build: [build-commonjs]         ╵                                 ~~~~~~~~
@neo4j-cypher/language-support:build: [build-commonjs] 
@neo4j-cypher/language-support:build: [build-commonjs]   The module "./dist/antlr4.web.mjs" was not found on the file system:
@neo4j-cypher/language-support:build: [build-commonjs] 
@neo4j-cypher/language-support:build: [build-commonjs]     ../../node_modules/antlr4/package.json:64:18:
@neo4j-cypher/language-support:build: [build-commonjs]       64 │         "import": "./dist/antlr4.web.mjs",
@neo4j-cypher/language-support:build: [build-commonjs]          ╵                   ~~~~~~~~~~~~~~~~~~~~~~~
@neo4j-cypher/language-support:build: [build-commonjs] 
@neo4j-cypher/language-support:build: [build-commonjs]   You can mark the path "antlr4" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
@neo4j-cypher/language-support:build: [build-commonjs] 
@neo4j-cypher/language-support:build: [build-esm] ✘ [ERROR] Could not resolve "antlr4"
@neo4j-cypher/language-support:build: [build-esm] 
@neo4j-cypher/language-support:build: [build-esm]     src/signatureHelp.ts:6:32:
@neo4j-cypher/language-support:build: [build-esm]       6 │ import { ParseTreeWalker } from 'antlr4';
@neo4j-cypher/language-support:build: [build-esm]         ╵                                 ~~~~~~~~
@neo4j-cypher/language-support:build: [build-esm] 
@neo4j-cypher/language-support:build: [build-esm]   The module "./dist/antlr4.web.mjs" was not found on the file system:
@neo4j-cypher/language-support:build: [build-esm] 
@neo4j-cypher/language-support:build: [build-esm]     ../../node_modules/antlr4/package.json:64:18:
@neo4j-cypher/language-support:build: [build-esm]       64 │         "import": "./dist/antlr4.web.mjs",
@neo4j-cypher/language-support:build: [build-esm]          ╵                   ~~~~~~~~~~~~~~~~~~~~~~~
```

I think we want to tell it you should build antlr4 and antlr4-c3 before language-support always, and also antlr4-c3 does need antlr4.

Reply:
You can also see it in the CI failed unit tests job

Reply:
Ah it seems you're right for our vendored deps, wonder why the automatic "build dependencies first" doesn't work for those.. Anyway, I left in the manual spec for those two in now :+1:

Reply:
Hopefully we shouldn't need to use `clean` or `--force` 🙏